create-release.py

```diff
@@ -420,6 +426,8 @@ def prepare_extensions_release() -> None:
         setup_file = os.path.join(extension_source_dir, 'setup.py')
         sed(setup_file, "{{package-name}}", extension)
         sed(setup_file, "{{version}}", config.new_version)
+        sed(setup_file, "{{data-files}}", "('share/jupyter/lab/extensions', glob('./dist/*.tgz'))")
```

Comment:
Should this be an absolute path?  Is this relative to `config.work_dir`?

Reply:
this is config relative to the "project structure", and it is hard coded. it's just that npm extensions need it, and python ones don't so I either set it or not.